dss-api.yml

```diff
@@ -208,6 +230,24 @@ definitions:
     required:
       - uuid
       - versions
+  bundle_version:
+    type: object
+    properties:
+      uuid:
+        type: string
+        description: Bundle unique ID
+      bundle_timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of bundle creation measured in milliseconds since epoch
+      contents:
+        type: array
+        items:
+          $ref: "#/definitions/file_version"
+      writer_uid:
```

Comment:
I don't like `created_by`, since there's ambiguity about the data type (name?  uid?).  I'll change it to `creator_uid`.